main/src/main/java/com/sedmelluq/discord/lavaplayer/source/nico/NicoAudioSourceManager.java

```diff
@@ -47,20 +47,24 @@ public class NicoAudioSourceManager implements AudioSourceManager, HttpConfigura
 
     private static final Pattern trackUrlPattern = Pattern.compile(TRACK_URL_REGEX);
 
-    private final String email;
-    private final String password;
     private final HttpInterfaceManager httpInterfaceManager;
     private final AtomicBoolean loggedIn;
 
+    public NicoAudioSourceManager() {
+        this(null, null);
+    }
+
     /**
      * @param email    Site account email
      * @param password Site account password
      */
     public NicoAudioSourceManager(String email, String password) {
-        this.email = email;
-        this.password = password;
         httpInterfaceManager = HttpClientTools.createDefaultThreadLocalManager();
         loggedIn = new AtomicBoolean();
+        // Log in at the start
```

Comment:
is niconico even usable without auth?

Reply:
Yes, I don't have an account and tested it without one.